app/enterprise/0.32-x/upgrades-migrations.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Upgrades and Migrations
+---
+
+# Upgrades and Migrations
+## Upgrades of Kong EE
+
+We define an upgrade, the process of changing the version of the software currently deployed to a more recent one, there is no difference with concept of update; normally associated with an upgrade between minor versions.
+
+We suggest to follow the below steps to perform an effective upgrade, this approach will require downtime, in case you need high availability review the Blue/Green Deployment section.
+
+1. Download the destination version of the product you are interest to upgrade to in Bintray
+2. Take a backup copy of the production database
+3. Review the [changelog](https://docs.konghq.com/enterprise/changelog/) for the destination version of the upgrade
+4. Prepare the preproduction environment with the copy of the database taken from production
+5. Install the newer package in preproduction.
+6. Apply any breaking changes to your preproduction environment
+7. Run the command kong migrations up
+8. Start Kong and validate the upgrade in preproduction
+9. Schedule the maintenance window for the upgrade in production
+10. Backup your production database in case of rollback.
+11. Install the package
+12. Adapt your infrastructure, kong configuration file and eventual custom nginx template according with your previous experience in preproduction.
+13. Run the kong migrations up command and start kong
+
+Once a new version is available you will be advised by a distribution list email or via your designated Customer Success Engineer (Premium subscription only). The newer version will be available in the same repository in Bintray.
```

Comment:
Please don't mention Bintray, and instead refer generically with something like "The new version will be available to download" with "download" linked to Bintray.

Reply:
Yes, eventually we should - but that isn't necessary as part of this PR.